### ISSUE TITLE
KeyError: '*skeleton'

### ISSUE THREAD

**Issue by mupgarlic** [BrynnaDaRosa/GTAIV-Anim-Importer#1]
(I'm new into modding, sorry if the issue is actually me, lmao)
I tried to import a .onim file into another model for gtaiv (a random pedestrian [.smd file]) and I got this message in the Blender Log:

KeyError: '*skeleton'

I have the strong feeling that I'm making something wrong, but I've spent a few hours trying with different rigged models and animations and I still get no results.

**Comment by BrynnaDaRosa** [comment]
.smd export from SparkIV is actually completely broken. It exports a skeleton of sorts but absolutely no vertex groups on your pedestrian model :( but try renaming the *skeleton bone to Char. 

Keep in mind this... Barely works. Extremely floaty animations (IK issue) and no exporting whatsoever. The only thing that seems to import correctly is facial animations and *some* gestures.

**Comment by DanielVolt** [comment]
Shouldn't close issues when an issue is still pending, it's there for a reason.

Although, i managed to fix some stuff and it imports it fine now and actually flushes the animation keys. Although it still needs work, please use more debug lines in the actual python file until it's stable (I added important ones).  

Feel free to look at my fork.

**Comment by Nirala-pri** [comment]
Is there any other way i can import ped models in blender?

**Comment by BrynnaDaRosa** [comment]
> Is there any other way i can import ped models in blender?

No. Currently, you will need an .fbx of a model exported out of Max that was imported via GIMS IV or OFIO.

**Comment by Nirala-pri** [comment]
sorry for being irreIevant, i was wondering if you can give me any rigged gta 4 character, i dont have 3ds max and i really need to use this script :/
